### PR TITLE
Hide API V1 after onboarding is made.

### PR DESCRIPTION
**Pull request by lauramelos**
This hide the API V1 keys on setting when the V2 onboarding is done.

Requested by Gerardo in Slack

### CODE REVIEW

File: includes/class-wc-gateway-amazon-payments-advanced.php [hunk 1]
```diff
@@ -2301,11 +2301,11 @@ public function filter_customer_field( $value ) {
 	 * Maybe hide standard WC checkout button on the cart, if enabled
 	 */
 	public function maybe_hide_standard_checkout_button() {
-		if ( ! $this->is_available() ) {
+		if ( ! $this->is_available() || $this->has_other_gateways_enabled() ) {
```
Comment (msaggiorato) [inline includes/class-wc-gateway-amazon-payments-advanced.php:2304]: Why this change? Seems unrelated to the settings.

**Review by msaggiorato — CHANGES_REQUESTED** [review]

File: includes/class-wc-gateway-amazon-payments-advanced.php (hunk 1 above)
Comment (lauramelos) [reply includes/class-wc-gateway-amazon-payments-advanced.php:2304]: yes, I mixed up on the cherry pick and rebase. Let me fix it and force push both open prs

**Review by msaggiorato — APPROVED** [review]
